src/main/java/com/wanderersoftherift/wotr/abilities/AbilityEvents.java

```diff
@@ -28,6 +31,16 @@
 @EventBusSubscriber(modid = WanderersOfTheRift.MODID, bus = EventBusSubscriber.Bus.GAME)
 public class AbilityEvents {
 
+    @SubscribeEvent
+    public static void registerAbilitySlots(ModifierHelper.IterateEquipmentSlotsEvent event) {
```

Comment:
This isn't really a registration event, since it is called every time equipment slots are being iterated. Perhaps we should have a WotrEquipmentSlot type registry though?